### ISSUE TITLE
Add option for HTML Output

### ISSUE THREAD

**Issue by cschroeder** [Widen/jira-changelog#5]
Currently, the changelog output can be either markdown or json.  It would be nice to offer raw html output for immediate publishing to a web page.

**Comment by uriahcarpenter** [comment]
I'd suggest the option of bundling a Markdown engine and running the output though that (vs. generating 'native' HTML).

**Comment by rnicholus** [comment]
@uriahcarpenter Yes, this was my initial thought: markdown -> HTML (or any other standardized format)

**Comment by dbeg** [comment]
.md -> * (anything)

:+1:
